src/components/facet-browser/intelligent-facets-search.graphql

```diff
@@ -0,0 +1,12 @@
+query intelligentFacets ($q: SearchQuery!, $facetsLimit: Int!, $valuesLimit: Int!) {
```

Comment:
Please move this file so it is co-located with there it is used.

This file is placed in the `/src/components/facet-browser/` directory but it is used by code in the `
src/components/facet-line/` directory.